src/pages/cli/graphql/authorization-rules.mdx

```diff
@@ -321,6 +321,28 @@ In the example above:
 - **any signed in user** is allowed to read the list of employees' `name` and `email` fields
 - **only the employee/owner themselves** have CRUD access to their `ssn` field
 
+<Callout warning>
+
+To prevent unintended loss of data, the user or role that attempts to `delete` a record should have delete permissions on every field of the `@model` annotated GraphQL type.
+For example, in the schema below: 
+```graphql
+type Todo @model @auth(rules: [
+  { allow: private, provider: iam }, 
+  { allow: groups, groups: ["Admin"] }
+])
+{
```

Comment:
```suggestion
]) {
```

minor formatting nit to align with other code examples 
![image](https://user-images.githubusercontent.com/5033303/208719162-1f53455a-6778-4737-80e1-33291cbbf905.png)